tests/unit/packaging/test_init.py

```diff
@@ -169,10 +170,10 @@ def key_factory(keystring, iterate_on=None, if_attr_exists=None):
     ]
 
     if with_bq_sync:
-        # assert (
-        #    pretend.call(crontab(minute=0), sync_bigquery_release_files)
-        #    in config.add_periodic_task.calls
-        # )
+        assert (
+            pretend.call(crontab(minute="*"), sync_bigquery_release_files)
```

Comment:
I'm unsure about running this every minute, primarily since if an individual batch takes longer than 1 minute, we will re-run that batch (since processed will still be False if the task hasn't completed and thus committed).

Thoughts on making processed a nullable, and setting processed to False when a file is added to a batch, then True once processing is complete?

This would allow us to initiate batches by looking for processed = None, and monitor for failures of batches if processed = False on completion of the backfill.